lab04_example/include/lab04_example/key_control.h

```diff
@@ -0,0 +1,54 @@
+/* Author: Luke Dennis l.dennis@unsw.edu.au */
+
+#include <ros/ros.h>
+#include <termios.h>
+
+namespace KeyControl {
+    /* https://github.com/sdipendra/ros-projects/blob/master/src/keyboard_non_blocking_input/src/keyboard_non_blocking_input_node.cpp*/
+    char getch() {
+        fd_set set;
```

Comment:
what is fd_set?

Reply:
This is code pulled from somewhere else (credited above) and waaay outside the scope of this course